### PR TITLE
Fix offline player error in /playtime for versions < 1.15.2

### LINKED ISSUE #4642: Error on /playtime <offlinePlayer>
### Type of bug

Error in console

### `/ess dump all` output

https://essentialsx.net/dump.html?id=7f2f908676b946c885610826126a4991

### Error log (if applicable)

https://pastebin.com/9iku0pSb

### Bug description

When I type "/playtime <offlinePlayer>", it doesn't output anything, just error in console.

### Steps to reproduce

Just "/playtime <offlinePlayer>"

### Expected behaviour

I wound need to get output of offline player playtime

### Actual behaviour

Nothing happens, just error in console

### PR DESCRIPTION
**Pull request by JRoy**
Fixes #4642

1.15.2 was the version which moved `getStatistic` to OfflinePlayer

### CODE REVIEW

File: Essentials/src/main/java/com/earth2me/essentials/commands/Commandplaytime.java [hunk 1]
```diff
@@ -33,7 +34,11 @@ protected void run(Server server, CommandSource sender, String commandLabel, Str
                 final IUser user = getPlayer(server, sender, args, 0);
                 displayName = user.getDisplayName();
                 playtime = user.getBase().getStatistic(PLAY_ONE_TICK);
-            } catch (PlayerNotFoundException ignored) {
+            } catch (PlayerNotFoundException e) {
+                // The ability to get the statistics of offline players is only available in 1.15.2+
+                if (VersionUtil.getServerBukkitVersion().isLowerThan(VersionUtil.v1_15_2_R01)) {
+                    throw e;
```
Comment (pop4959) [inline Essentials/src/main/java/com/earth2me/essentials/commands/Commandplaytime.java:40]: Just curious, what's the user facing message that gets sent for this?

Comment (JRoy) [reply Essentials/src/main/java/com/earth2me/essentials/commands/Commandplaytime.java:40]: > Just curious, what's the user facing message that gets sent for this?

the standard player could not be found message

Comment (pop4959) [reply Essentials/src/main/java/com/earth2me/essentials/commands/Commandplaytime.java:40]: Ok, figured, just wanted to check

**Review by pop4959 — APPROVED** [review]

File: Essentials/src/main/java/com/earth2me/essentials/commands/Commandplaytime.java (hunk 1 above)
Comment (LudnicaKiller) [reply Essentials/src/main/java/com/earth2me/essentials/commands/Commandplaytime.java:40]: @pop4959 no message, just error in console:

26.11 17:13:55 [Server] ERROR Command playtime failed:
26.11 17:13:55 [Server] INFO java.lang.NoSuchMethodError: org.bukkit.OfflinePlayer.getStatistic(Lorg/bukkit/Statistic;)I
26.11 17:13:55 [Server] INFO at com.earth2me.essentials.commands.Commandplaytime.run(Commandplaytime.java:39) ~[EssentialsX.jar:?]
26.11 17:13:55 [Server] INFO at com.earth2me.essentials.commands.EssentialsCommand.run(EssentialsCommand.java:177) ~[EssentialsX.jar:?]
26.11 17:13:55 [Server] INFO at com.earth2me.essentials.commands.EssentialsCommand.run(EssentialsCommand.java:172) ~[EssentialsX.jar:?]
26.11 17:13:55 [Server] INFO at com.earth2me.essentials.Essentials.onCommandEssentials(Essentials.java:777) [EssentialsX.jar:?]
26.11 17:13:55 [Server] INFO at com.earth2me.essentials.Essentials.onCommand(Essentials.java:680) [EssentialsX.jar:?]
26.11 17:13:55 [Server] INFO at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) [patched.jar:git-PaperSpigot-"4c7641d"]
26.11 17:13:55 [Server] INFO at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:146) [patched.jar:git-PaperSpigot-"4c7641d"]
26.11 17:13:55 [Server] INFO at org.bukkit.craftbukkit.v1_8_R3.CraftServer.dispatchCommand(CraftServer.java:666) [patched.jar:git-PaperSpigot-"4c7641d"]
26.11 17:13:55 [Server] INFO at net.minecraft.server.v1_8_R3.PlayerConnection.handleCommand(PlayerConnection.java:1189) [patched.jar:git-PaperSpigot-"4c7641d"]
26.11 17:13:55 [Server] INFO at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:1001) [patched.jar:git-PaperSpigot-"4c7641d"]
26.11 17:13:55 [Server] INFO at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:45) [patched.jar:git-PaperSpigot-"4c7641d"]
26.11 17:13:55 [Server] INFO at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:1) [patched.jar:git-PaperSpigot-"4c7641d"]
26.11 17:13:55 [Server] INFO at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) [patched.jar:git-PaperSpigot-"4c7641d"]
26.11 17:13:55 [Server] INFO at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_292]
26.11 17:13:55 [Server] INFO at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_292]
26.11 17:13:55 [Server] INFO at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) [patched.jar:git-PaperSpigot-"4c7641d"]
26.11 17:13:55 [Server] INFO at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:774) [patched.jar:git-PaperSpigot-"4c7641d"]
26.11 17:13:55 [Server] INFO at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:378) [patched.jar:git-PaperSpigot-"4c7641d"]
26.11 17:13:55 [Server] INFO at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:713) [patched.jar:git-PaperSpigot-"4c7641d"]
26.11 17:13:55 [Server] INFO at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:616) [patched.jar:git-PaperSpigot-"4c7641d"]
26.11 17:13:55 [Server] INFO at java.lang.Thread.run(Thread.java:748) [?:1.8.0_292]

Comment (pop4959) [reply Essentials/src/main/java/com/earth2me/essentials/commands/Commandplaytime.java:40]: This is already fixed, download the latest build. If something still isn't working for some reason please make a new bug report.

**Pull request merged**